docs/writing-policies/spec/host-capabilities/05-crypto.md

```diff
@@ -28,18 +28,29 @@ performing cryptographic checks exposed by the host:
 <td>
 
 ```hcl
+# Certificate:
+{
+  # **mandatory**: Which encoding is used by the certificate
+  # Either the string "Pem" or "Der"; case sensetive.
+  "encoding": string,
+  # Actual certificate
+  # The certificate is UTF-8 encoded.
+  # Each byte is a UTF-8 rune.
+  "data": [byte(int), ..., byte(int)]
+}
+
 {
   # **mandatory**: PEM-encoded certificate to verify
-  "certificate": string,
+  "cert": Certificate,
   # optional:
   "cert_chain": [
       # list of PEM-encoded certs, ordered by trust
       # usage (intermediates first, root last)
```

Comment:
This is a list of Certificate objects, as defined before. Is that right? In that case we can drop the `PEM` string, because a `Certificate` object can be both PEM and DER